src/main/resources/jenkins/plugins/logstash/LogstashConfiguration/help-enableGlobally.html

```diff
@@ -1 +1 @@
-Checking will make all non pipeline builds forward their log to the above indexer.
+Checking will make all builds forward their log to the above indexer.
```

Comment:
I don't think this is true, i.e, old style (= non pipeline) builds will not be affected

Reply:
You mean that previous version's right, is it?

I found that checking `Enable Globally` true also applies to the pipeline builds.

Jenkins version: 2.348
logstash plugin: 2.5.0205

pipeline script:
```
pipeline {
    agent any

    stages {
        stage('Hello') {
            steps {
                build 'log-test'
            }
        }
    }
}
```
with log-test job's `Send console log to Logstash` unabled.

Would it be okay to commit above description then?

Reply:
Sorry, I misread this. I think your version is correct